contracts/CollateralTracker.sol

```diff
@@ -1495,7 +1495,7 @@ contract CollateralTracker is ERC20Minimal, Multicall {
         // Offset determined whether to consider the last tokenId from the list
         // (a potentially newly minted position)
         uint256 totalIterations = positionBalanceArray.length;
-        for (uint256 i = 0; i < totalIterations; ) {
```

Comment:
Please revert. Although this is logically identical, we explicitly specify =0 here for clarity.